app/src/main/res/values/arrays.xml

```diff
@@ -32,6 +32,14 @@
         <item>0</item>
         <item>5</item>
     </integer-array>
+    <string-array name="systemClockNames" translatable="false">
+        <item>System clock</item>
```

Comment:
Shouldn't this perhaps be Device clock (Like you said in the description)?

Reply:
Makes sense. Will be changed in the next commit.